ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -135,8 +135,15 @@ private static void handleFailure(final RoutingContext routingContext) {
       log.error(String.format("Failed to handle request %d %s", routingContext.statusCode(),
           routingContext.request().path()),
           routingContext.failure());
+      ServerUtils.handleError(
+          routingContext.response(),
+          routingContext.statusCode(),
+          routingContext.statusCode(),
```

Comment:
We should use an error code (creating a new one if necessary) from the ErrorCodes class here, not the HTTP status code.